### PR TITLE
Including code points in warning for missing glyphs in fonts

### PR DESCRIPTION
**Pull request by Lucas-C**
Implementing suggestion from https://github.com/py-pdf/fpdf2/issues/1171#issuecomment-2126845201.
Also slightly improving a docstring as mentioned there: https://github.com/py-pdf/fpdf2/issues/1156#issuecomment-2095444563

**Checklist**:

- [x] The GitHub pipeline is OK (green), meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.

- [x] A unit test is covering the code added / modified by this PR

- [x] This PR is ready to be merged

- [x] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder

- [NA] A mention of the change is present in `CHANGELOG.md`

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/py-pdf/fpdf2/blob/master/LICENSE).

### CODE REVIEW

File: fpdf/output.py [hunk 1]
```diff
@@ -553,7 +553,10 @@ def _add_fonts(self):
                     LOGGER.warning(
                         "Font %s is missing the following glyphs: %s",
                         fontname,
-                        ", ".join(chr(x) for x in font.missing_glyphs),
+                        ", ".join(
+                            f"{chr(x)} ({chr(x).encode('unicode-escape').decode()})"
+                            for x in font.missing_glyphs
```
Comment (gmischler) [inline fpdf/output.py:558]: Imagine someone trying to render a lengthy chinese text with a latin font. That is going to be a multi-page log entry...

Maybe we should limit the output to the first 10 missing glyphs, or so?

File: fpdf/output.py [hunk 2]
```diff
@@ -553,7 +553,10 @@ def _add_fonts(self):
                     LOGGER.warning(
                         "Font %s is missing the following glyphs: %s",
                         fontname,
-                        ", ".join(chr(x) for x in font.missing_glyphs),
+                        ", ".join(
+                            f"{chr(x)} ({chr(x).encode('unicode-escape').decode()})"
```
Comment (gmischler) [inline fpdf/output.py:557]: Maybe put some kind of quotes around the literal character, to make it obvious when it is a non-printing or whitespace character?

**Review by gmischler — APPROVED** [review]

File: fpdf/output.py (hunk 1 above)
Comment (Lucas-C) [reply fpdf/output.py:558]: Excellent idea!
I just implemented that.

File: fpdf/output.py (hunk 2 above)
Comment (Lucas-C) [reply fpdf/output.py:557]: Excellent idea!
I just implemented that.

**Pull request merged**